products/BellHybrid/apps/application-bell-onboarding/ApplicationBellOnBoarding.cpp

```diff
@@ -174,9 +174,11 @@ namespace app
 
         informationPromptTimer.stop();
 
+        // TODO gerPrevWindow is removed for good reasons :)
+        // this has to be fixed with @Przemek
```

Comment:
yea yea -> will get back here after rebase -> that file has been refactored and changes are not here.